R/writeManifest.R

```diff
@@ -1,9 +1,18 @@
-#' Create a manifest.json describing deployment requirements.
+#' Create a `manifest.json` for Git-backed content
```

Comment:
Manifests can be useful when using the APIs, when using `rsconnect-python` (to deploy R content), or when using Git-backed content.

Let's not imply that this is only used by Git workflows.